pkg/scheduler/api/devices/nvidia/gpushare/device_info.go

```diff
@@ -168,6 +168,10 @@ func (gs *GPUDevices) FilterNode(pod *v1.Pod) (int, string, error) {
 	return devices.Success, "", nil
 }
 
+func (gs *GPUDevices) ScoreNode(pod *v1.Pod) float64 {
```

Comment:
Why not implement score logic here but deviceScore plugin?

Reply:
We need a plugin to register in "BatchNodeOrder", to invoke these "score" method.